mappings/net/minecraft/world/biome/BiomeDecorator.mapping

```diff
@@ -0,0 +1,57 @@
+CLASS net/minecraft/class_122 net/minecraft/world/biome/BiomeDecorator
+	FIELD field_455 treesPerChunk I
+	FIELD field_456 flowersPerChunk I
+	FIELD field_457 grassPerChunk I
+	FIELD field_458 deadBushesPerChunk I
+	FIELD field_459 mushroomsPerChunk I
+	FIELD field_460 sugarcanePerChunk I
+	FIELD field_461 cactusPerChunk I
+	FIELD field_462 gravelDisksPerChunk I
+	FIELD field_463 sandDisksPerChunk I
+	FIELD field_464 clayPerChunk I
+	FIELD field_465 hugeMushroomsPerChunk I
+	FIELD field_466 generateLakes Z
+	FIELD field_467 world Lnet/minecraft/class_99;
+	FIELD field_468 random Ljava/util/Random;
+	FIELD field_469 startPos Lnet/minecraft/class_1372;
+	FIELD field_470 worldProperties Lnet/minecraft/class_417;
+	FIELD field_471 clayFeature Lnet/minecraft/class_441;
+	FIELD field_472 sandDiskFeature Lnet/minecraft/class_441;
+	FIELD field_473 gravelDiskFeature Lnet/minecraft/class_441;
+	FIELD field_474 dirtFeature Lnet/minecraft/class_441;
+	FIELD field_475 gravelFeature Lnet/minecraft/class_441;
+	FIELD field_476 graniteFeature Lnet/minecraft/class_441;
```

Comment:
are these not final? if not final these are fine but if they are capitalize

Reply:
they are lateinit instance fields